ide-migration/server/test/Module.js

```diff
@@ -57,14 +55,14 @@ var pathModule = createInternalModule("path", function (exports) {
       if (directory === "." && prev !== undefined) continue;
 
       if (
-        directory === ".."
-        && directories.length
-        && prev !== ".."
-        && prev !== undefined
-        && (prev !== "" || keepBlanks)
+        directory === ".." &&
+        directories.length &&
```

Comment:
why not `and` before each variable?

Reply:
Sadly, this could not be configured in Prettier.